packages/paste-website/src/pages/patterns/privacy/index.mdx

```diff
@@ -0,0 +1,286 @@
+- Refer to PII as “personal data”, where “PII/non-PII” terminology is reserved for internal use.
+- Provide a subtle prompt about personal data to users, rather than a warning.
+- Are different from a [required field indicator](/components/input/#required-field-indicator), which is a visual representation to inform a user of a required field.
+
+### Design
+
+A privacy indicator is [Help Text](/components/help-text/#accessibility) and has specific copy to inform a user of a non-PII field. Within the Help Text is an Anchor linking out to [Twilio docs on how we handle PII](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii).
+
+<LivePreview scope={{Label, Input, HelpText, Anchor}}>{PrivacyExample}</LivePreview>
+
+### Accessibility
+
+Because the privacy indicator leverages [Help Text](/components/help-text/#accessibility), it shares all of the Accessibility guidelines of that component.
+
+## Variations
+
+### Friendly or unique name
+
+Use when informing a user that a “friendly name” or "unique name” field is non-PII.
+
+<LivePreview scope={{Label, Input, HelpText, Anchor, Button, Heading, Box}}>{FriendlyName}</LivePreview>
+
+<Callout>
+  <CalloutTitle>Need a variation that isn't included here?</CalloutTitle>
+  <CalloutText>
+    Variations of this pattern will be added as there are more non-PII fields where users may unintentionally input
+    personal data. Please{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions" showExternal>
+      create a GitHub Discussion
+    </Anchor>{' '}
+    if there is a variant you would like added.
+  </CalloutText>
+</Callout>
+
+## Composition
+
+### Positioning with additional help text
+
+If there is additional help text on a form component, append the privacy help text after the additional help text.
+
+<LivePreview scope={{Label, Input, HelpText, Anchor}}>{WithHelpText}</LivePreview>
+
+### Positioning with error text
+
+If an input containing a privacy indicator is in an error state, the error message will replace <strong> all </strong> help text, including the privacy indicator.
+
+<LivePreview scope={{Label, Input, HelpText, Anchor}} noInline>
+  {WithErrorText}
+</LivePreview>
+
+## Dos and don'ts
+
+<DoDont>
+  <Do title="Do" body="Use the same copy as in the pattern to indicate a non-PII field." preview={false}>
+    <Box padding="space60">
+      <Label htmlFor="do-1">Friendly name</Label>
+      <Input type="text" id="do-1" aria-describedby="do-1-ht" value="My favorite service" />
+      <HelpText id="do-1-ht">
+        Avoid entering{' '}
+        <Anchor
+          href="https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii"
+          showExternal
+        >
+          personal data
+        </Anchor>{' '}
+        like your name, email address, or phone number.
+      </HelpText>
+    </Box>
+  </Do>
+  {/* omitted the anchor to GH Discussions in the body here, is that ok? */}
```

Comment:
There was an anchor in the spec on this "Don't" body text, let me know if there's a way to do that in the body attribute. (I already tried creating a separate `dontBody` template literal in another folder and importing that in to use as the `body` value but that didn't work)

Reply:
Looking at the DoDon't component, the body prop is just a string, doesn't look like there's a way to support html content currently.